easybuild/easyblocks/p/pgi.py

```diff
@@ -0,0 +1,94 @@
+##
+# Copyright 2015 Bart Oldeman
```

Comment:
usually, copyright on contributions is transferred to Ghent University

if you'd like to retain copyright on this yourself, please read https://github.com/hpcugent/easybuild-easyblocks/blob/master/CONTRIBUTING.md and modify this bit accordingly